06-classification_continued.Rmd

```diff
@@ -540,17 +638,17 @@ training observations. We can evaluate a classifier by splitting the data
 randomly into a training and test data set, using the training set to build the
 classifier, and using the test set to estimate its accuracy. To tune the
 classifier (e.g., select the K in K-nearest neighbours), we maximize accuracy
-estimates from cross-validation.
+estimates from cross-validation. This process is summarized in Figure \@ref(fig:06-overview).
 
 ```{r 06-overview, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Overview of K-nn classification", fig.retina = 2, out.width = "660"}
 knitr::include_graphics("img/train-test-overview.jpeg")
 ```
 
 The overall workflow for performing K-nearest neighbour classification using `tidymodels` is as follows:
```

Comment:
These numbered steps do not correspond to the other 6 numbered steps above... Maybe we shouldn't number the ones above?

Reply:
done